locale/pt-br/texts.po

```diff
@@ -1744,11 +1814,23 @@ msgstr "${ tokenSymbol } Depósito"
 msgid "${ tokenSymbol } Withdrawal"
 msgstr "${ tokenSymbol } Saque"
 
-#: src/components/Reown/NanoContract/NanoContractActions.js:101
+#: src/components/Reown/NanoContract/NanoContractActions.js:38
+msgid "${ tokenSymbol } Grant Authority"
+msgstr ""
```

Comment:
chore: Missing translation
```suggestion
msgid "${ tokenSymbol } Grant Authority"
msgstr "${ tokenSymbol } Conceder autoridade"
```

Reply:
I don't like the idea of translating those... I left them blank on purpose, just like we don't translate "Mint Authority" and "Melt Authority"